### ISSUE TITLE
(🐞) `dmypy inspect` doesn't support windows drive paths `c:\`

### ISSUE THREAD

**Issue by KotlinIsland** [python/mypy#15780]
`C:\test.py`
```py
a: int
```
```
👉 dmypy run C:\test.py --export-types
Success: no issues found in 1 source file
👉 dmypy inspect C:\test.py:1:1
Format should be file:line:column[:end_line:end_column]
```
